package/gluon-scheduled-domain-switch/files/usr/bin/gluon-check-gw-reachability

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+GW_COUNT=$(batctl gwl -H 2> /dev/null | wc -l)
```

Comment:
this does work exclusively for batman. The IBSS => 802.11s switch *might* be relevant or other mesh protocols too.
OTOH it is unlikely that someone builds a new network with more recently supported mesh protocol.

Reply:
As only one mesh protocol can be selected at this time a migration between them is impossible. Correct me if I'm wrong.

Reply:
Fixed: https://github.com/freifunk-gluon/gluon/pull/1555/commits/82df55ea0049736cd0ade23ea5fa31c621ad13d4#diff-0219e17e415d8777b3329280a95a57a5R11